apps/daffio/src/app/content/docs/api-docs/containers/api-docs-list/api-docs-list.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+import { DaffioDocService } from '../../services/api-doc.service';
+import { DaffioApiDocReference } from '../../models/api-doc-reference';
+
+@Component({
+  selector: 'daffio-api-docs-list-container',
+  templateUrl: './api-docs-list.component.html'
+})
+export class ApiDocsListContainer implements OnInit {
+
+  /**
+   * A list of references for API documents.
+   */
+  docsList: DaffioApiDocReference[];
+
+  constructor(
+    private docService: DaffioDocService
+  ) {}
+
+  ngOnInit() {
+    this.docService.getDocsList().subscribe(
```

Comment:
You need to cleanup the subscription, or use the async pipe in the template.

Reply:
used an async pipe instead of subscription